### PR TITLE
Update duplicati.rb

### PR DESCRIPTION
**Pull request by daylightbrightledlight**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

I've added some Duplicati zaps.

### CODE REVIEW

**Comment by daylightbrightledlight** [comment]
For some reason, the CI build could not compete.

File: Casks/duplicati.rb [hunk 1]
```diff
@@ -9,4 +9,9 @@
   homepage 'https://www.duplicati.com/'
 
   app 'Duplicati.app'
+  
+  zap delete: [
```
Comment (commitay) [inline Casks/duplicati.rb]: `zap trash`

**Review by commitay — CHANGES_REQUESTED** [review]

**Comment by commitay** [comment]
@daylightbrightledlight Please add PR templates and complete `style` and `audit` checks for all open PRs.

**Comment by daylightbrightledlight** [comment]
@commitay Hi there. I've read the [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#testing-your-new-cask), but I am having trouble trying to configure brew to use my patched casks. How do I edit the cask using my repository without messing up brew cask from being able to work normally?

**Comment by daylightbrightledlight** [comment]
> Please add PR templates

I'll add the PR templates.

**Comment by commitay** [comment]
> but I am having trouble trying to configure brew to use my patched casks. How do I edit the cask using my repository without messing up brew cask from being able to work normally?

You don't need to configure anything.

Run `brew cask style --fix /path/to/edited/cask.rb` and `brew cask audit --download /path/to/edited/cask.rb`

**Comment by daylightbrightledlight** [comment]
@commitay Okay thanks. I used `brew cask style --fix`. After `Installing or updating 'rubocop-cask' gem` finished, I ended up with the following error:

```
E: 25:  1: unexpected token kEND
(Using Ruby 2.1 parser; configure using TargetRubyVersion parameter, under AllCops)

1 file inspected, 1 offense detected
Error: style check failed
```

I'll work on trying to fix this and keep you updated.

**Comment by commitay** [comment]
> However the audit failed:

```
 - appcast checkpoint mismatch
Expected: a62964fa18b2896ce5de9ac6c7a2012457c1bfb4b36d9cf2ae4f336aca5e65fd
Actual: 1d9839044f40dd3ee88a76d1074c8c34b7162a0960512932deeaa1e2d94fe9ee
```

You need to replace the `appcast` checkpoint with its current value.

File: Casks/duplicati.rb [hunk 2]
```diff
@@ -9,4 +9,9 @@
   homepage 'https://www.duplicati.com/'
 
   app 'Duplicati.app'
+
+  zap delete: [
```
Comment (commitay) [inline Casks/duplicati.rb]: `zap trash`

Comment (daylightbrightledlight) [reply Casks/duplicati.rb]: Okay. I've changed it to `zap trash`

**Comment by daylightbrightledlight** [comment]
I think I've completed the requirements. Please let me know if this patch is ready.

**Review by commitay — APPROVED** [review]

**Pull request merged**